src/core/query.ts

```diff
@@ -112,7 +112,11 @@ export class Query {
   }
 
   withoutSorts(...sorts: Sort[]): Query {
-    this.request.sort = this.request.sort.filter((oldSort) => !sorts.find((sort) => sort.field === oldSort.field));
+    this.request.sort = this.request.sort.filter((oldSort) => !sorts.find((sort) =>
```

Comment:
This feels inefficient because it's an O(n<sup>2</sup>) algorithm. We can bring it down to O(n) by going through `sorts` and adding each string representation of `ids` as keys to an `ids` object and each `field` as keys to a `fields` object, then going through `this.request.sort` and looking them up in the appropriate objects.

That said, `sorts` (and possibly `this.request.sort`) is probably going to be small enough (less than 5?) that we can probably get away with what we're already doing.